app/home/views/dashboard.html

```diff
@@ -33,14 +33,14 @@ <h3 class="panel-title">{{'USER_DASHBOARD_FREQUENT_ACTIONS' | translate}}</h3>
                 </div>
             </div>
     </div>
-</div>-->
+</div>
 
-<!--<div class="panel panel-primary content-border-margin">
+<div class="panel panel-primary content-border-margin">
```

Comment:
since we're still in development a mocked up functionality (prototype) can give an idea of how the functionality looks like, specially when discussing with users. I think it doesn't harm to have it there for now and we can remove for production.